api/mod/fixtures/test_data.json

```diff
@@ -1,7 +1,7 @@
 {
     "alembic_version": [
         {
-            "version_num": "981ce91575bc"
+            "version_num": "be60cc3fc1d3"
```

Comment:
Sounds good!

Reply:
Im not sure because this could be information for alembic to know up to which migration the database state is currently in sync (did that make sense?). Anyway, maybe we should look up the docs what this table is for, to prevent any database created from the test data to be un-migrateable.